libs/ledger-live-common/src/families/cosmos/api/Cosmos.ts

```diff
@@ -85,6 +93,12 @@ export class CosmosAPI {
     }
   };
 
+  private getCosmosSDKVersion = async (): Promise<string> => {
+    const { application_version } = await this.getNodeInfo();
```

Comment:
We do twice this call, right ? :thinking: maybe there is a way to store it so we can do only one call to nodeInfo, it is constant

Reply:
why do you think we do this call twice? 

I've save the "_cosmosSDKVersion" in a variable here
```
  private _cosmosSDKVersion: Promise<string> | null = null;
  private get cosmosSDKVersion(): Promise<string> {
    if (!this._cosmosSDKVersion) {
      this._cosmosSDKVersion = this.getCosmosSDKVersion();
    }
    return this._cosmosSDKVersion;
  }
```
so we call only once the getNodeInfo each time we sync an account.

it is true that we can save the node_info somewhere so we don't need to do that for each account that has the same currency.
but as the node_info api is very light and we may switch to the mintscan API in the near future to fetch transactions. I think it is a minor issue at this moment

Reply:
Yes I was talking about the node_info part :smile: , ok understood